gallery/visualtests/nxOverflowTooltip.js

```diff
@@ -27,7 +27,8 @@ describe('NxOverflowTooltip', function() {
       maxBeforeOverflowStr = 'WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW',
       descendantOverflowItemSelector = `${descendantOverflowListSelector} .nx-list__item:nth-child(2)`;
 
-  const wait1Sec = () => wait(1000);
+  // NOTE: not really 1 sec anymore
+  const wait1Sec = () => wait(1500);
```

Comment:
> I don't think it's that important, but do you think we should update the name because it's no longer 1 sec.

That reminds me, I meant to lower the batch timeout.  I don't think it needs to be a whole second, since it's just the amount of time to wait for stragglers.  All units of work resulting from an initial page render should come in at essentially the same time.  Done in  0ecb3a9dc.  This means that this function name is now truthful again.